modules/end-user-guide/partials/proc_adding-a-vs-code-extension-to-the-che-plugin-registry.adoc

```diff
@@ -0,0 +1,79 @@
+    command:                                                     <13>
+        - /bin/sh
+    args:                                                        <14>
+        - "-c"
+        - "./entrypoint.sh"
+    volumeMounts:                                                <15>
+      - name: vscode-ext-volume                                  <16>
+        path: "/home/theia/my-vscode-ext"                        <17>
+    endpoints:                                                   <18>
+      - name: "configuration-endpoint"                           <19>
+        public: true                                             <20>
+        targetPort: 61436                                        <21>
+        attributes:                                              <22>
+          protocol: http
+  extensions:                                                    <23>
+          - https://github.com/redhat-developer/vscode-yaml/releases/download/0.4.0/redhat.vscode-yaml-0.4.0.vsix
+          - https://github.com/SonarSource/sonarlint-vscode/releases/download/1.16.0/sonarlint-vscode-1.16.0.vsix
+----
+<1> (OPTIONAL) The ID of the plugin, useful if a plugin has multiple entries for one repository (for example, Java 8 vs. Java 11)
+<2> Repository information about the plugin. If ID is specified then this field is not a list element.
+<3> The URL to the extension's git repository URL
+<4> Tag or SHA1 ID of the upstream repository that hosts the extension, usually corresponding to a version/snapshot/release
+<5> (OPTIONAL) An alias for this plugin: this means anything listed here will get its own meta.yaml generated
+<6> (OPTIONAL) If the plugin runs in a sidecar container, then the sidecar information is specified here
+<7> A location of a container image to be used as this plugin's sidecar. This line cannot be specified concurrently with `directory` (see above).
+<8> (OPTIONAL) The name of the sidecar container
+<9> (OPTIONAL) The memory limit of the sidecar container
+<10> (OPTIONAL) The memory request of the sidecar container
+<11> (OPTIONAL) The CPU limit of the sidecar container
+<12> (OPTIONAL) The CPU request of the sidecar container
+<13> (OPTIONAL) Definitions of root process commands inside container
+<14> (OPTIONAL) Arguments for root process commands inside container
+<15> (OPTIONAL) Any volume mounting information for the sidecar container
+<16> The name of the mount
+<17> The path of the mount
+<18> (OPTIONAL) Any endpoint information for the sidecar container
+<19> Endpoint name
+<20> Whether or not the endpoint is exposed publically or not
+<21> The port number
+<22> Attributes relating to the endpoint
+<23> Direct link(s) to the vsix files included with this plugin. The vsix built by the repository specified (i.e. the "main extension) must be listed first
+
+. Run the `build.sh` script with the options of your choosing. The build process will generate `meta.yaml` files automatically, based on the entries in the `che-theia-plugins.yaml` file.
```

Comment:
I am not sure if it is related to the PR but build failed for me
```
[test] Summary of all failing tests
[test]  FAIL  tests/sidecar/sidecar-docker-image.spec.ts
  ● Test Sidecar › basics
[test] 
    TypeError: Object.fromEntries is not a function

      47 |       n: '1',
      48 |     };
    > 49 |     const result = await this.git.log(logOptions);
         |                                   ^
      50 |     const latest = result.latest;
      51 |     if (!latest) {
      52 |       throw new Error(`Unable to find result when executing ${JSON.stringify(logOptions)}`);
[test] 
      at userOptions (../../node_modules/simple-git/src/lib/tasks/log.ts:63:18)
      at parseLogOptions (../../node_modules/simple-git/src/lib/tasks/log.ts:102:22)
[test]       at Git.Object.<anonymous>.Git.log (../../node_modules/simple-git/src/git.js:871:26)
      at SidecarDockerImage.<anonymous> (src/sidecar/sidecar-docker-image.ts:49:35)
[test]       at src/sidecar/sidecar-docker-image.ts:1824:41
[test]       at Object.<anonymous>.__awaiter (src/sidecar/sidecar-docker-image.ts:1767:10)
[test]       at SidecarDockerImage.getDockerImageFor (src/sidecar/sidecar-docker-image.ts:1878:12)
[test]       at tests/sidecar/sidecar-docker-image.spec.ts:32:45
[test] 
[test]  FAIL  tests/registry/registry-helper.spec.ts
[test]   ● Test RegistryHelper › basics with current sha1
[test] 
[test]     TypeError: Object.fromEntries is not a function
[test] 
[test]       39 |       n: '1',
[test]       40 |     };
[test]     > 41 |     const result = await this.git.log(logOptions);
[test]          |                                   ^
[test]       42 |     // eslint-disable-next-line @typescript-eslint/no-explicit-any
[test]       43 |     const hash = (result.latest as any).hash;
[test]       44 |     this.shortSha1 = hash.substring(0, 7);
[test] 
[test]       at userOptions (../../node_modules/simple-git/src/lib/tasks/log.ts:63:18)
[test]       at parseLogOptions (../../node_modules/simple-git/src/lib/tasks/log.ts:102:22)
[test]       at Git.Object.<anonymous>.Git.log (../../node_modules/simple-git/src/git.js:871:26)
[test]       at RegistryHelper.<anonymous> (src/registry/registry-helper.ts:41:35)
      at fulfilled (src/registry/registry-helper.ts:2196:24)
[test] 
[test]  FAIL  tests/che-theia-plugin/che-theia-plugin-analyzer.spec.ts
[test]   ● Test CheTheiaPluginsAnalyzer › basics
[test] 
[test]     TypeError: Cannot read property 'url' of undefined
[test] 
[test]       44 | 
[test]       45 |     // search without id as it is not provided
[test]     > 46 |     const goPlugins = result.plugins.filter(plugin => plugin.repository.url === 'https://github.com/golang/vscode-go');
[test]          |                                                                         ^
[test]       47 |     expect(goPlugins).toBeDefined();
[test]       48 |     expect(goPlugins.length).toBe(1);
[test]       49 |     const goPlugin = goPlugins[0];
[test] 
[test]       at result.plugins.filter.plugin (tests/che-theia-plugin/che-theia-plugin-analyzer.spec.ts:46:73)
[test]           at Array.filter (<anonymous>)
[test]       at tests/che-theia-plugin/che-theia-plugin-analyzer.spec.ts:46:38
[test]       at fulfilled (tests/che-theia-plugin/che-theia-plugin-analyzer.spec.ts:5:58)
[test] 
[test] 
[test] Test Suites: 3 failed, 25 passed, 28 total
[test] Tests:       3 failed, 93 passed, 96 total
[test] Snapshots:   0 total
[test] Time:        8.109 s, estimated 13 s
[test] Ran all test suites.
[test] Force exiting Jest: Have you considered using `--detectOpenHandles` to detect async operations that kept running after all tests finished?
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
[test] yarn test exited with code 1
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Reply:
probably old nodejs version ?